eras/conway/impl/cddl-files/conway.cddl

```diff
@@ -556,30 +556,37 @@ script_pubkey = (0, addr_keyhash)
 script_all = (1, [ * native_script ])
 script_any = (2, [ * native_script ])
 script_n_of_k = (3, n: uint, [ * native_script ])
-invalid_before = (4, uint)
-invalid_hereafter = (5, uint)
+invalid_before = (4, slot_no)
+invalid_hereafter = (5, slot_no)
 
 coin = uint
 
 multiasset<a> = { + policy_id => { + asset_name => a } }
 policy_id = scripthash
 asset_name = bytes .size (0..32)
 
-negInt64 = -9223372036854775808 .. -1
-posInt64 = 1 .. 9223372036854775807
+minInt64 = -9223372036854775808
+maxInt64 = 9223372036854775807
+maxWord64 = 18446744073709551615
+
+negInt64 = minInt64 .. -1
+posInt64 = 1 ..maxInt64
```

Comment:
```suggestion
posInt64 = 1 .. maxInt64
```
Perhaps it would make it clearer? The documentation on `ranges` states that `.`s are included in names, so it would help to introduce a ` ` (space) to disambiguate?